lls/main.c

```diff
@@ -389,14 +409,19 @@ lls_proc(void *arg)
 
 	while (likely(!exiting)) {
 		/* Read in packets on front and back interfaces. */
-		int num_tx = process_pkts(lls_conf, front,
-			lls_conf->rx_queue_front, lls_conf->tx_queue_front);
-		if ((num_tx > 0) && lacp_enabled(net_conf, front)) {
-			if (lacp_timer_reset(lls_conf, front) < 0)
-				RTE_LOG(NOTICE, TIMER, "Can't reset front LACP timer to skip cycle\n");
+		int num_tx;
+
+		if (front->hw_filter_eth) {
```

Comment:
Isn't it possible that a NIC does not support Ethernet filter and sends all non-IP packets to queue zero?

If so, the test in this `if` statement is not correct. And the same problem happens to the back interface.